docs/docs/starters.md

```diff
@@ -68,6 +68,60 @@ Official starters are maintained by Gatsby.
 | [gatsby-starter-blog](https://github.com/gatsbyjs/gatsby-starter-blog)               | [Demo](https://gatsby-starter-blog-demo.netlify.com/)        | Create a basic blog            | Blog post pages and listings |
 | [gatsby-starter-hello-world](https://github.com/gatsbyjs/gatsby-starter-hello-world) | [Demo](https://gatsby-starter-hello-world-demo.netlify.com/) | Learn Gatsby                   | Gatsby bare essentials       |
 
+## 🧐 What's inside?
+
+A quick look at the top-level files and directories you'll see in a Gatsby project.
```

Comment:
```suggestion
A quick look at common top-level files and directories you'll see in a Gatsby project.
```

You could also link to the Gatsby Project Structure doc instead: https://www.gatsbyjs.org/docs/gatsby-project-structure/